src/ape/logging.py

```diff
@@ -24,7 +24,7 @@ class LogLevel(IntEnum):
 logging.addLevelName(LogLevel.SUCCESS.value, LogLevel.SUCCESS.name)
 logging.SUCCESS = LogLevel.SUCCESS.value  # type: ignore
 DEFAULT_LOG_LEVEL = LogLevel.INFO.name
-DEFAULT_LOG_FORMAT = "%(levelname)s%(plugin)s: %(message)s"
+DEFAULT_LOG_FORMAT = "%(levelname)-7s%(plugin)s: %(message)s"
```

Comment:
```suggestion
DEFAULT_LOG_FORMAT = "%(levelname)s%(plugin)-7s: %(message)s"
```

Reply:
I think that the format you suggested aligns the plugin name but not the log level. If  there is a plugin, the logs will not be properly aligned. I suggest using the following format:
DEFAULT_LOG_FORMAT = "%(levelname)-7s %(plugin)-15s: %(message)s"
I set the plugin field to 15 spaces since I noticed that some plugins have long names. 
What do you think?